### PR TITLE
Update domains.json

### PR DESCRIPTION
**Pull request by Builder0x**
Added line 14
"vote.refluid.net": "refluid.eth",

#### Hi! What is your PR about?
- Add a custom domain for your space

### CODE REVIEW

File: spaces/domains.json [hunk 1]
```diff
@@ -11,6 +11,7 @@
   "vote.protectorate.xyz": "protectoratexyz.eth",
   "vote.hk.chat": "hkchat.eth",
   "vote.daocred.xyz": "daocred.eth",
+  "vote.refluid.net": "refluid.eth",
```
Comment (ChaituVR) [inline spaces/domains.json:14]: I see 
image

Any idea?

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 1 above)
Comment (Builder0x) [reply spaces/domains.json:14]: Sloved, Please check again

Comment (ChaituVR) [reply spaces/domains.json:14]: image
Not working too

Looks like there is no CNAME at all 
image

**Review by ChaituVR — CHANGES_REQUESTED** [review]

File: spaces/domains.json (hunk 1 above)
Comment (Builder0x) [reply spaces/domains.json:14]: oh sorry
We added the CNAME
Please check again

**Review by ChaituVR — APPROVED** [review]

**Pull request merged**